webpack/assets/javascripts/react_app/components/hosts/powerStatus/PowerStatus.scss

```diff
@@ -0,0 +1,11 @@
+.host-power-status {
+  &.on {
+    color: #3f9c35;
```

Comment:
i've moved powerStatus files to their own dedicated directory inside hosts, thats the relation